gallery/src/pages/Home.tsx

```diff
@@ -24,35 +27,157 @@ const Home = () =>
+          <NxDescriptionList.Description>
+            Components which make up the "page skeleton" are included in this section. This includes page-level components
+            such as the global sidebar and also containers such as tiles and accordions. In addition, rich examples of
+            layouts incorporating multiple components working together are also found here.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Typography and Icons</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes semantic and typographic containers for text content, such as paragraphs and code
+            blocks.  It also includes utility components that facilitate the use of icons.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>HTML Variants</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Some RSC components and styles have separate support for usage via React component and via bare CSS classes.
+            In these cases, the React components and bare CSS classes are documented separately. For any such component
+            which has both types of documentation, the HTML documentation is currently sequestered in this section.
+            Generally, for React-using consumers, the React components should be preferred over the bare CSS class
+            approach, and this section should thus generally be ignored.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Mixins and Helpers</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes CSS and SCSS utilities such as helper classes, SCSS mixins, and CSS variables.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>JavaScript and TypeScript Utilities</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            This section includes utilities that can be useful when writing code that uses RSC components. No actual React
+            components are present in this section, however React hooks are documented here in addition to JavaScript
+            functions and TypeScript types.
+          </NxDescriptionList.Description>
+        </NxDescriptionList.Item>
+        <NxDescriptionList.Item>
+          <NxDescriptionList.Term>Guidelines</NxDescriptionList.Term>
+          <NxDescriptionList.Description>
+            Rather than documentation of specific components or usable pieces of code, this section includes higher level
+            descriptions, tutorials, and recommendations regarding usage of RSC components and web UI development in
+            general.
```

Comment:
This section covers general documentation on web UI development, and tutorials, recommendations for using RSC components.